MoreLinq/From.cs

```diff
@@ -38,7 +38,9 @@ public static IEnumerable<T> From<T>(Func<T> function)
         {
             return _(); IEnumerable<T> _()
             {
+#pragma warning disable CA1062 // Validate arguments of public methods
                 yield return function();
```

Comment:
Is there a reason not to check that `function` is not null?

Reply:
See the discussion in PR #333 for background on this.